packages/st2/debian/postinst

```diff
@@ -44,6 +44,10 @@ set_permissions() {
   echo "$fileperms" | cut -f1,2,4 -d' ' | xargs -L1 chmod
 }
 
+register_runners() {
+    st2ctl reload --register-runners
```

Comment:
Curious, won't this fail on st2 package upgrade if Mongo connection is not available?